src/labels/CustomLabels.labels

```diff
@@ -6871,6 +6871,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Common error when a total doesn't match Donation Amount.</shortDescription>
         <value>Total amount doesn't match the {0}</value>
     </labels>
+    <labels>
+        <fullName>geErrorBatchGiftEntryReauthRequired</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when payment attributes changed on authorized gift</shortDescription>
+        <value>To update donor, date or amount, reenter card details to reauthorized the payment.</value>
```

Comment:
@melissabarber - FYI, here's a new label that I'll be using for an error message here.

Reply:
Thanks! added one suggestion for the shortDescription other than that, all good to go.